src/lib.rs

```diff
@@ -197,38 +201,8 @@ impl<AC: AdditionalClaims> OidcClient<AC> {
         //TODO remove borrow with next breaking version
```

Comment:
Maybe we should do that in this PR ? (remove the ref)

Reply:
Yes!